.devcontainer/Dockerfile

```diff
@@ -2,5 +2,15 @@ FROM ghcr.io/containerbase/node:18.16.0@sha256:fc48ac97eb5d0dee207d4a336a52d7d3b
 
 USER root
 
+WORKDIR /workspace
+COPY . .
+
```

Comment:
```suggestion
```
doesn't make sense, because you want to run on the mounted host repo, not a static copy from build

Reply:
Those instructions are needed for the devcontainer workflow. They do not overrule the mounting when devcontainer is spinned locally.

Reply:
they mount the local system when running the container

https://github.com/devcontainers/ci/blob/de7272670e54f2b3b63137c14b031561799bbec5/common/src/docker.ts#L276

renovate source should not copied to image.

Reply:
Thank you for sharing the link. I was not able to find this mount point.